gold-cc-cvc-input.html

```diff
@@ -63,8 +67,9 @@
         display: block;
       }
 
-      iron-icon {
+      .cvc-icon {
```

Comment:
For some reason when I put the mixin inside the `iron-icon` tag with the shadow dom on the mixin wasn`t applied, so I put a class to fix this behavior.